01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,9 @@
+for (let num = 1; num <= 20; num++) {
+  if (num % 15 === 0) {
+    console.log("FizzBuzz");
+  } else if (num % 5 === 0) {
+    console.log("Buzz");
+  } else if (num % 3 === 0) {
+    console.log("Fizz");
+  } else console.log(num);
```

Comment:
```suggestion
  } else {
    console.log(num);
  }
```

にしなかった理由って何かありますか？（なんでここだけ？と思ってしまいました）

Reply:
コメントありがとうございます。
特に理由はなく忘れていただけです。
修正いたしました。